### PR TITLE
add San Benito County

### PR DESCRIPTION
**Pull request by ben-hatcher**

### CODE REVIEW

File: scrapers.js [hunk 1]
```diff
@@ -575,7 +575,8 @@ let scrapers = [
     // Incapsula blocking request
     scraper: async function() {
       let counties = [];
-      let $ = await fetch.headless(this.url);
+      let $ = await fetch.page(this.url);
```
Comment (lazd) [inline scrapers.js]: Accidentally changed?

File: scrapers.js [hunk 2]
```diff
@@ -1158,25 +1196,23 @@ let scrapers = [
         $('.count-subject:contains("Positive travel-related case")')
           .closest('.hb-counter')
           .find('.count-number')
-          .attr('data-from')
+          .text()
```
Comment (lazd) [inline scrapers.js]: Accidentally changed?

**Comment by lazd** [comment]
Can you rebase and clean the accidental changes out? Thanks!

**Comment by ben-hatcher** [comment]
will do!

**Comment by lazd** [comment]
Hey Ben, I adapted your code and got it in!

**Pull request closed without merging**